packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/go/changeInsideLambdaBat.yml

```diff
@@ -0,0 +1,28 @@
+languageId: go
+command:
+  version: 6
+  spokenForm: change inside lambda bat
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - {type: interiorOnly}
+        - type: containingScope
+          scopeType: {type: anonymousFunction}
+      mark: {type: decoratedSymbol, symbolColor: default, character: b}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: _ = func() { /* body */ }
```

Comment:
Wow lambdas really do look similar to named functions in Go. I wonder if we should just make "funk" a superset of "lambda" in Go

Reply:
I like that. I'll do it.

I would really like to set up a per-language skeleton (https://github.com/cursorless-dev/cursorless/issues/1642), so I can start adding notes to it as I go. I'm a bit concerned about the perfect (autogenerated! list of all scopes! etc.) being the enemy of the good here, and forgetting useful details along the way.

I'll mark this PR as to-discuss for that purpose.

Reply:
Done, with a simple test. If/when this merges, will update the wiki.

Reply:
> Wow lambdas really do look similar to named functions in Go. I wonder if we should just make "funk" a superset of "lambda" in Go

@AndreasArvidsson any objections to this one? Technically inconsistent with other languages but if you look at the syntax they're very similar to names functions

@josharian is it common to pass a short anonymous function directly as a function call arg? That's the main place where we prefer not to get trapped by "funk" in javascript

Reply:
>  is it common to pass a short anonymous function directly as a function call arg?

Not super common, but not rare either.

To my mind the most important aspect is that IMHO it is rare to be looking at a piece of code and not know whether it is in an anonymous func or not. It's a combination of syntax and usage. And typescript it is common for code in lambdas to outnumber the rest of the code by a significant factor, and to have that fact be relatively locally invisible.

There is long-standing discussion about adding more concise lambdas to the language, e.g. https://github.com/golang/go/issues/21498. If that went in, I can definitely see the desire to use `lambda` for them.

Reply:
But I don't feel super strongly.

Reply:
If it's hard to tell them apart I would be perfectly fine with supporting funk. It would be really irritating if you had to think about the syntax when using cursorless.

Reply:
Cool sounds like we're on the same page then

Reply:
> I would really like to set up a per-language docs page skeleton (#1642), so I can start adding notes to it as I go. I'm a bit concerned about the perfect (autogenerated! list of all scopes! etc.) being the enemy of the good here, and forgetting useful details along the way.
> 
> I'll mark this PR as to-discuss for that purpose.

I'm not sure we should have this PR be to-discuss for that purpose, as that seems outside the scope of this PR. Prob better to mark that issue to-discuss and merge this PR